serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -20,13 +20,13 @@ image::getting-started/hello-world-workflow.png[]
 .Prerequisites
 * Java {java_min_version} is installed with `JAVA_HOME` configured appropriately.
 * Apache Maven {maven_min_version} is installed.
-* {quarkus_cli_url}[Quarkus CLI] {quarkus_version} or xref:tooling/kn-plugin-workflow-overview.adoc[Knative CLI] {kn-cli-version} is installed.
+* {quarkus_cli_url}[Quarkus CLI] or xref:tooling/kn-plugin-workflow-overview.adoc[Knative CLI] {kn-cli-version} is installed.
```

Comment:
Should we say Kn Workflow CLI here?  In next version the plan is to bundle them together, but currently you need to explicitly install the kn-workflow extension, where this might give the impression it's just Kn CLI (although clicking the link obviously takes you to the page where it is properly explained).

Reply:
Good point! I'll update it.